src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -76,14 +198,29 @@ public ServiceCollection(Accessor as, String path) {
     // */
     /**
      * This method configures a set of services.
-     * @param key Property Name
-     * @param value Property Value
+     * @param key name of service route
+     * @param value source of service route
      * @param subject Value of the service subject
      * @throws DaoException Exception thrown
      */
     public void configure(String key, String value, String subject) throws DaoException {
         RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
-        rest.setService(this.getPath(), key, value, subject);
+        ArrayList<IPersoniumServiceRoute> routes = new ArrayList<>();
+        routes.add(new PersoniumServiceRoute(key, value));
+        String configureXML = createConfigureXML(routes, subject);
+        rest.proppatch(this.getPath(), configureXML);
+    }
```

Comment:
To avoid DRY, could you change this codes to the following?

```java
ArrayList<IPersoniumServiceRoute> routes = new ArrayList<>();
routes.add(new PersoniumServiceRoute(key, value));
configure(routes, subject);
```